### ISSUE TITLE
[packager] JReleaser error without details

### ISSUE THREAD

**Issue by aalmiray** [jreleaser/jreleaser#1234]
### Discussed in https://github.com/jreleaser/jreleaser/discussions/1233

<div type='discussions-op-text'>

<sup>Originally posted by **gotson** March  3, 2023</sup>
Hello,

my JReleaser release on CI failed with no details whatsoever: https://github.com/gotson/komga/actions/runs/4322026365/jobs/7544068802#step:11:838

```
INFO]  JReleaser 1.5.0
[INFO]    - basedir set to /home/runner/work/komga/komga/komga
[INFO]    - outputdir set to /home/runner/work/komga/komga/komga/build/jreleaser
[INFO]  git-root-search set to true
[INFO]  Loading variables from /home/runner/.jreleaser/config.properties
[WARN]  Variables source /home/runner/.jreleaser/config.properties does not exist
[INFO]  Validating configuration
[INFO]  Strict mode set to false
[INFO]    [validation] Adding registry docker.io to distribution.komga.docker with username '***'
[INFO]  Project version set to 0.162.0
[INFO]  Release is not snapshot
[INFO]  Timestamp is 2023-03-03T09:31:51.906Z
[INFO]  HEAD is at cb332a6
[INFO]  Platform is linux-x86_64
[INFO]  dry-run set to false
[INFO]  Generating changelog
[INFO]  Extracting issues from changelog
[INFO]  Storing issues: build/jreleaser/release/issues.txt
[INFO]  Storing changelog: build/jreleaser/release/CHANGELOG.md
[INFO]  Storing changelog: ../CHANGELOG.md
[INFO]  Cataloging artifacts
[INFO]    [sbom] Cataloging is not enabled. Skipping
[INFO]  Calculating checksums for distributions and files
[INFO]    [checksum] build/libs/komga-0.162.0.jar.sha256
[INFO]  Signing distributions and files
[INFO]    [sign] Signing is not enabled. Skipping
[INFO]  Deploying Maven artifacts
[INFO]    [maven] Deploying is not enabled. Skipping
[INFO]  Uploading distributions and files
[INFO]    [upload] Uploading is not enabled. Skipping
[INFO]  Releasing to https://github.com/***/komga
[INFO]   - uploading checksums_sha256.txt
[INFO]   - uploading komga-0.162.0.jar
[INFO]   - uploading komga-0.162.0.jar.sha256
[WARN]  Could not update release v0.162.0 with discussion category Announcements
[INFO]  Marking 2 issue(s) as released
[INFO]  Preparing distributions
[INFO]    - Preparing komga distribution
[INFO]      [docker] preparing komga distribution
[INFO]  Packaging distributions
[INFO]    - Packaging komga distribution
[INFO]      [docker] packaging komga distribution
[INFO]      [docker]  - docker.io/***/komga:latest
[INFO]      [docker]  - docker.io/***/komga:0.162.0
[INFO]      [docker]  - docker.io/***/komga:0.x
[INFO]      [docker]  - ghcr.io/***/komga:latest
[INFO]      [docker]  - ghcr.io/***/komga:0.162.0
[INFO]      [docker]  - ghcr.io/***/komga:0.x
[INFO]  Publishing distributions
[INFO]    - Publishing komga distribution
[INFO]      [docker] publishing komga distribution
[INFO]      [docker] setting up repository ***/komga-docker
[INFO]    Announcing release
[INFO]      [announce] Announcing is not enabled. Skipping
[INFO]  Writing output properties to build/jreleaser/output.properties
[ERROR] JReleaser failed after 01:00 m
```

Any idea what happened ?

The release was created on Github, the files were attached.

However the Docker images were not pushed to the registries, and the closed issues were not commented upon,</div>

**Comment by aalmiray** [comment]
The value of `packagers.docker.repository.active` is not propagated to `distributions.<name>.docker.registry.active`.

**Comment by aalmiray** [comment]
🎉 This issue has been resolved in `v1.5.1` ([Release Notes](https://github.com/jreleaser/jreleaser/releases/tag/v1.5.1))
